Microsoft.Azure.Cosmos/src/Diagnostics/CosmosDiagnostics.cs

```diff
@@ -23,6 +23,18 @@ public virtual TimeSpan GetClientElapsedTime()
             throw new NotImplementedException($"CosmosDiagnostics.GetElapsedTime");
         }
 
+        /// <summary>
+        /// This represent the start time of the request.
```

Comment:
```suggestion
        /// This represent the start time of the operation.
```

This is the global Diagnostics object, the user performs an operation, the operation might span multiple requests.